### ISSUE TITLE
enh: clarify the user experience for Paused resources

### ISSUE THREAD

**Issue by acpana** [GoogleCloudPlatform/k8s-config-connector#1398]
When `kccmanager` is `Paused` via `CC/ CCC` values for `actuationMode` the only indication that a resources is not being actuated onto the cloud provider today are the log line (todo ref). There's a few proposal to enhance the user experience:

#### Add ConfigConnector pause events 

this is mostly captured in #1335 to add Config Connector [events](https://cloud.google.com/config-connector/docs/how-to/monitoring-your-resources#config-connector-events)

#### Use CC/ CCC status

We can use the CC/ CCC objects, which today are required to be present in order for `Pause` mode to be activated., to write to the `Status` subresources of CC/ CCC objects. (todo ref)

####  Use the health probes

Another way to let operators know that resources will not get actuated at a a glance is to introduce modify the health probe for CC/ CCC to reflect that a cluster/ namespace is paused.

**Comment by jingyih** [comment]
Based on my understanding, if we use pause events, we are surfacing "pause" per resource object. And if we use CC/CCC status, we are surfacing "pause" per cluster/namespace?

Do you have an (fake) example of what the health probes look like?

**Comment by acpana** [comment]
> Based on my understanding, if we use pause events, we are surfacing "pause" per resource object. And if we use CC/CCC status, we are surfacing "pause" per cluster/namespace?

Correct. I added a note that the options are not mutually exclusive. Indeed we can choose to do them all!

> Do you have an (fake) example of what the health probes look like?

@cheftako can keep me honest here but IIUC we could add a `paused` value to the `status.Healthy` fields of CC/ CCC. Currently I think it's `true` if reconciliation (of a CC/CCC res) succeeds, false otherwise.

**Comment by ldanielmadariaga** [comment]
This item has been open for >12 months. As part of scheduled repository maintenance, this Issue/PR will be closed. If this is still a critical ask, please re-open and work with your Google Cloud Account team to prioritize it with GCP KCC Product Management.
